cli/cli.go

```diff
@@ -284,6 +290,31 @@ func fetchIndex() {
 	}
 }
 
+func installFromVersionFile() {
+	blob, err := ioutil.ReadFile(VERSION_FILE)
+	if err != nil {
+		fmtc.Println("Cannot read %s", VERSION_FILE)
+		exit(1)
+	}
+
+	version, err := readFirstWord(string(blob))
+	if err != nil {
+		fmtc.Println("Cannot find version in %s", VERSION_FILE)
+		exit(1)
+	}
+
+	fmtc.Println("Installing version %s from %s", version, VERSION_FILE)
+	installCommand(version)
+}
+
+func readFirstWord(body string) (string, error) {
+	matches := regexp.MustCompile(`^\s*(\S+)\s*`).FindStringSubmatch(body)
+	if len(matches) < 2 {
+		return "", errors.New("I have no words!")
```

Comment:
You should return error with text which you use in other place of your code. If you don't use error text, you should return bool value instead of error.